dists/showmethekey.desktop.in

```diff
@@ -4,8 +4,8 @@ Name=Show Me The Key
 GenericName=Show Me The Key
 Comment=Show keys you typed on screen.
 Icon=showmethekey
-Exec=showmethekey-gtk
-TryExec=/usr/bin/showmethekey-gtk
+Exec=@install_prefix@/@package_bindir@/showmethekey-gtk
+TryExec=@install_prefix@/@package_bindir@/showmethekey-gtk
```

Comment:
This should be configured with package_bindir being an absolute path calculated by meson using `join_paths()`, because Meson 0.62.0 allows specifying a bindir that is outside of the prefix (useful for e.g. NixOS).

Reply:
ff967cd4832b0685260c1a66ad9a3442a8cb7b26

Reply:
I’m not entirely sure I understand you correctly, please look at 1dc36b02b9519a0f48a7415948f551719f29363b.

Reply:
Yes, that is what I meant.